ansible/group_vars/all

```diff
@@ -435,8 +442,9 @@ metrics:
 
 user_events: "{{ user_events_enabled | default(false) | lower }}"
 
-durationChecker:
-    timeWindow: "{{ duration_checker_time_window | default('1 d') }}"
+zeroDowntimeDeployment:
+  enabled: "{{ zerodowntime_deployment_switch | default(true) }}"
+  solution: "{{ zerodowntime_deployment_solution | default('apicall') }}"
```

Comment:
It seems `apicall` is not readable here, how about use `scroll` or other readable words? anyway, it means `scroll to deploy`.

Another deployment solution is `half`, which means `a small blue/green deployment solution`

Reply:
Currently, this configuration does not take effect due to some missing parts.
I believe we can address what you mentioned in the subsequent PR when we introduce a feature for zero-downtime deployment.

Reply:
Got it.